docs/configuring-playbook-appservice-draupnir-for-all.md

```diff
@@ -0,0 +1,93 @@
+# Setting up Draupnir for All/D4A (optional)
+
+The playbook can install and configure the [Draupnir](https://github.com/the-draupnir-project/Draupnir) moderation tool for you in appservice mode.
+
+Appservice mode can be used together with the regular [Draupnir bot](configuring-playbook-bot-draupnir.md) or independently.
+
+D4A compared to Draupnir Normal. 
```

Comment:
Should you be calling this "normal"? Is that what it's called internal to mdad? it's probably better to call it "bot mode" so it's consistent with the way Draupnir's documentation refers to the same thing.